lib/LANraragi/Plugin/Metadata/Hitomi.pm

```diff
@@ -0,0 +1,253 @@
+package LANraragi::Plugin::Metadata::Hitomi;
```

Comment:
It would be nice to add tests for this plugin before integrating it! It helps a lot for monitoring status of metadata providers so we can stay on top when they change formats. (Or add cloudflare 😿)  

There are essentially three test types for plugins:  
* Dead simple test on the namespace to check if it's interpreted by Perl without any issues: This goes in https://github.com/Difegue/LANraragi/blob/dev/tests/modules.t
* Basic tests on the live website, which are basically just calling your `get_tags_from_hitomi` method with a known good gallery ID and verifying that we get the proper tags out of it: See https://github.com/Difegue/LANraragi/blob/dev/tests/plugins.t for examples.
* Offline tests on a sample of data, so that we can always test the plugin logic even if the live website is unavailable. In this case the only sample you'd have to provide would be one of those gigantic hitomi JSONs. See https://github.com/Difegue/LANraragi/tree/dev/tests/samples for the sample folder, and https://github.com/Difegue/LANraragi/tree/dev/tests/LANraragi/Plugin/Metadata for examples.  

I won't ask you to do the third type if it sounds too daunting to you, but it'd certainly make my life easier. 😅  
The first two should be pretty easy though!